### PR TITLE
fix(console): node title ignores the user defined resource name

### PR DESCRIPTION
**Pull request by ainvoner**
resolves: https://github.com/winglang/wing/issues/5994

for the following code:
```
bring cloud;
class A {
  new() {
    nodeof(this).title = "MyTitle";
    nodeof(this).color = "violet";
  }
}
class B {
    new() {
    }
}
new A();
new cloud.Bucket() as "MyBucket";
new cloud.Bucket();
new B();
```

the console will display the following:
Screenshot 2024-04-11 at 12 54 26

## Checklist

- [ ] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [ ] Description explains motivation and solution
- [ ] Tests added (always)
- [ ] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

**Comment by ainvoner** [comment]
@ekeren please take a look.

**Comment by ekeren** [comment]
Great!!!

**Comment by ekeren** [comment]
I'll let someone else code review

File: apps/wing-console/console/ui/src/ui/resource-metadata.tsx [hunk 1]
```diff
@@ -283,6 +283,13 @@ export const ResourceMetadata = memo(
       };
     }, [node, inbound, outbound]);
 
+    const nodeLabel = useMemo(() => {
+      const cloudResourceTypeName = node.type.split(".").at(-1) || "";
+      const compilerNamed =
```
Comment (ekeren) [inline apps/wing-console/console/ui/src/ui/resource-metadata.tsx:288]: Smart and simple 💪

**Review by ekeren — APPROVED** [review]

**Pull request merged**